adoc/deployment-bootstrap.adoc

```diff
@@ -3,13 +3,92 @@
 
 === Cluster Deployment
 
+==== Cluster initialization
 . Initialize the cluster on the deployed machines.
 As `--control-plane` enter the IP/FQDN of your load balancer. If you do not use a load balancer use your first master node.
 +
 ----
 skuba cluster init --control-plane <LB IP/FQDN> my-cluster
 ----
 `cluster init` generates the folder named `my-cluster` and initializes the directory that will hold the configuration for the cluster.
+
+==== Cluster configuration
+
+At this point you can perform some configurations before bootstrapping the cluster.
+
+===== Prevent nodes running special workloads from being rebooted
+
+. Open the `kured` deployment in `my-cluster/addons/kured/kured.yaml`
+. Adapt the `DaemonSet` by adding one of the following flags to the `command` section of the `kured` container:
++
+----
+---
+apiVersion: apps/v1
+kind: DaemonSet
+...
+spec:
+  ...
+    ...
+      ...
+      containers:
+        ...
+          command:
+            - /usr/bin/kured
+            - --blocking-pod-selector=name=<NAME OF POD>
+----
+
+You can add any key/value labels to this selector:
+----
+--blocking-pod-selector=<LABEL KEY 1>=<LABLE VALUE 1>,<LABEL KEY 2>=<LABEL VALUE 2>
+----
+
+Alternatively you can adapt the `kured` DaemonSet also later during runtime (after bootstrap) via:
+----
+kubectl edit daemonset kured --namespace kube-system
+----
+
+This will restart all `kured` pods with the additional configuration flags.
+
+==== Prevent nodes with any prometheus alerts from being rebooted
+
+[NOTE]
+====
+By default, **any** prometheus alert blocks a node from reboot. However you can filter specific alerts to be ignored via the `--alert-filter-regexp` flag.
+====
+
+. Open the `kured` deployment in `my-cluster/addons/kured/kured.yaml`
+. Adapt the `DaemonSet` by adding one of the following flags to the `command` section of the `kured` container:
++
+----
+---
+apiVersion: apps/v1
+kind: DaemonSet
+...
+spec:
+  ...
+    ...
+      ...
+      containers:
+        ...
+          command:
+            - /usr/bin/kured
+            - --prometheus-url=<PROMETHEUS SERVER URL>
+            - --alert-filter-regexp=^(RebootRequired|AnotherBenignAlert|...$
+----
+
+[IMPORTANT]
+====
+The <PROMETHEUS SERVER URL> needs to contain the protocol (`http://` or `https://`)
+====
+
```

Comment:
same here with `kubectl apply`

Reply:
here same thing, skuba will do the apply. the user should never have to `kubectl apply` - only maybe `kubectl edit`